app/__init__.py

```diff
@@ -229,12 +229,13 @@ def make_celery(app=None):
 
 
 # Health-check
-health = HealthCheck(current_app, "/health-check")
-health.add_check(health_check_celery)
+health = HealthCheck()
+health_dependencies = HealthCheck()
 health.add_check(health_check_db)
 with current_app.app_context():
     current_app.config['MIGRATION_STATUS'] = check_migrations()
 health.add_check(health_check_migrations)
+health_dependencies.add_check(health_check_celery())
```

Comment:
See what was written previously and what you have written

Reply:
@iamareebjamal Previously it was written 
`health.add_check(health_check_celery)` But how do we add to same instance of health-check when we want only celery result at a seperate endpoint.

Reply:
I have made a different instance to get the result of celery

Reply:
Again you are looking at a completely different thing than what I am trying to point out

```diff
-health.add_check(health_check_celery)
+health_dependencies.add_check(health_check_celery())
```

Reply:
Can you see any other difference than health and health_dependencies?

Reply:
@iamareebjamal Please read the comment sir. I am looking at the same line

Reply:
```
health.add_check(health_check_celery)
```

Reply:
@iamareebjamal I am at the same point and direction you said. Also I wrote why I did this, please read.

Reply:
Yes I saw. Sorry

Reply:
But it's a function, in python dont they have ( ) braces when calling a function. Dumb me. Apologies

Reply:
Writing kotlin and java daily had made me done this :( Will not repeat again. Thanks